noip.sh

```diff
@@ -3,12 +3,22 @@
 SOURCE_CONF=/config/noip.conf
 GENERATED_CONF=/config/no-ip2.generated.conf
 
-# Search for custom config file, if it doesn't exist, copy the default one
```

Comment:
Thanks for your pull request, but I'd rather support both methods -- config file and env var. (With the env var overriding the config file, if both are specified.)